Frontend/src/app/clusters/page.tsx

```diff
@@ -16,12 +20,38 @@ export default function Clusters() {
   const {
     token: { colorBgContainer, borderRadiusLG },
   } = theme.useToken();
+  const dispatch = useDispatch();
+  const { userData } = useAppSelector((state) => state.authData);
+  const [clusters, setClusters] = useState<IClusterDetails[]>([]);
+  const [filteredClusters, setFilteredClusters] = useState<IClusterDetails[]>([]);
 
-  const onSearch: SearchProps["onSearch"] = (value, _e, info) =>
-    console.log(info?.source, value);
+  const getAllCluster = async () => {
+    try{
+    const res = await getAllClusters(userData._id);
+    if(res.data){
+      setClusters(res.data)
+    }
+    }catch(err){
+      message.error("Failed to fetch clusters");
```

Comment:
Please put more details in the error message. What do you mean by cluster here?

Reply:
We can connect to multiple JasmineGraph servers through the UI, those are referred to as the cluster.

Reply:
Ok